apps/rush-lib/src/logic/PublishGit.ts

```diff
@@ -101,9 +102,9 @@ export class PublishGit {
     PublishUtilities.execCommand(!!this._targetBranch && shouldExecute, this._gitPath, [
       'tag',
       '-a',
-      tagName,
+      npmTag ? `${tagName}-${npmTag}` : tagName,
       '-m',
-      `${packageName} v${packageVersion}`,
+      npmTag ? `${packageName} v${packageVersion}-${npmTag}` : `${packageName} v${packageVersion}`,
```

Comment:
Instead of using the NPM tag, wouldn't it be better to use the real version? Like if the npm tag is "dev", but you publish versions `1.0.0-dev.0` and `1.0.0-dev1`, both with the `dev` tag, they'll both end up with the same Git tag.

Reply:
That parameter does have a confusing name. Yeah, it should be the prerelease name.